src/neo4j_genai/retrievers/hybrid.py

```diff
@@ -13,31 +13,32 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 from __future__ import annotations
-from typing import Optional, Any, Callable
+
+import logging
+from typing import Any, Callable, Optional
 
 import neo4j
 from pydantic import ValidationError
 
 from neo4j_genai.embedder import Embedder
 from neo4j_genai.exceptions import (
+    EmbeddingRequiredError,
     RetrieverInitializationError,
     SearchValidationError,
-    EmbeddingRequiredError,
 )
+from neo4j_genai.neo4j_queries import get_search_query
```

Comment:
Out of curiosity, which rule are you using to sort the imports? Is it something we should enforce for the whole team?

Reply:
I'm asking because it makes the review more difficult, if there is a convention there we can follow, happy to apply it.

Reply:
I'm just using Ruff. I have it set to do formatting and import sorting through the Python VSCode extension. Maybe we could set up the CI/CD to check the imports are sorted correctly as well as the formatting is correct?

Reply:
Ok, thanks. Yes, makes sense to me to add it to CI, and maybe in README so that we can run the check locally before pushing - but not for this PR.